tofu/main/aws/outputs.tf

```diff
@@ -29,7 +29,7 @@ locals {
     ingress_class_name   = module.k3s_cluster[i].ingress_class_name
     }
   }
-  rke2_outputs = { for i, cluster in local.rke2_clusters : cluster.name => {
+  rke2_outputs = { for i, cluster in local.rke2_clusters : cluster.name_prefix => {
```

Comment:
I understand this code did not work - but shouldn't `name_prefix` be renamed to `name` instead?

To me `name_prefix` makes sense for a cluster template, while a cluster (proper) should have a `name` instead.

How about, when we build clusters from templates (in aws/locals.tf) we turn `name_prefix`es in templates into `name`s in clusters?